File: examples/cache/increment/node/incrementAValue.js
import * as cache from "@m3o/services/cache";

// Increment a value (if it's a number)
async function IncrementAvalue() {
  let cacheService = new cache.CacheService(process.env.MICRO_API_TOKEN);
  let rsp = await cacheService.increment({
    key: "counter",
    value: 2,
  });
  console.log(rsp);
}

await IncrementAvalue();

File: examples/cache/increment/go/incrementAValue.go
package example

import (
	"fmt"
	"github.com/micro/services/clients/go/cache"
	"os"
)

// Increment a value (if it's a number)
func IncrementAvalue() {
	cacheService := cache.NewCacheService(os.Getenv("MICRO_API_TOKEN"))
	rsp, err := cacheService.Increment(&cache.IncrementRequest{
		Key:   "counter",
		Value: 2,
	})
	fmt.Println(rsp, err)
}
